backend/routes/recipes.py

```
@@ -209,8 +209,42 @@ def update_recipe(recipe_id):
+                    "error": detailed_error,
+                    "validation_details": error_details if error_details else [str(e)],
+                }
+            ),
+            400,
+        )
+    except Exception as e:
+        print(f"Unexpected error in update_recipe: {e}")
+        import traceback
+
+        traceback.print_exc()
+        return jsonify({"error": f"Failed to update recipe: {str(e)}"}), 500
```

Ruff (0.13.1)

Use explicit conversion flag

Replace with conversion flag

(RUF010)